src/Bravo3/Orm/Services/AbstractManagerUtility.php

```diff
@@ -87,4 +93,75 @@ protected function getEntityId($entity)
         $reader   = new Reader($metadata, $entity);
         return $reader->getId();
     }
+
+    /**
+     * Get the table name for a given entity class
+     *
+     * @param string $class
+     * @return string
+     */
+    protected function getTableForClass($class)
```

Comment:
this potentially could substitute our mappings system we've implemented, right?

https://github.com/news-life-media/phoenix/blob/master/src/phoenix/src/Phoenix/Resources/config/entities_mapping.yml

https://github.com/news-life-media/phoenix/blob/master/src/phoenix/src/Phoenix/Entity/Redis/AbstractEntity.php#L178